src/main/resources/logback.xml

```diff
@@ -18,9 +18,15 @@
     </encoder>
   </appender>
 
+  <appender name="Sentry" class="com.getsentry.raven.logback.SentryAppender">
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
```

Comment:
we can only filter on class level, so if set the level higher on a class we miss all other warnings/errors from that class too. 

do we really want to do it?